package.json

```diff
@@ -50,13 +52,31 @@
     "docs:watch": "gitbook install && gitbook serve",
     "prepublish": "nwb build",
     "release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci",
-    "test:ts": "tsfmt -r && tsc -p ./",
-    "test:watch": "nwb test --server"
+    "pretest": "npm run style && npm run test:node && npm run test:ts",
```

Comment:
also helps to reuse and execute the same hooks without repeating:

thanks to extracting `test` to `pretest` and `test` now we can run
```
"test:watch": "npm test -- --server",
```

which will run pretest, test with `--server`


before this wasn't possible without repeating. I would had to do: 
```
test: "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci"
test:watch": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test --server"
```